src/pages/studies/studies.tsx

```diff
@@ -15,7 +15,7 @@ let previousMappingData: any | null = null;
 
 const Studies: FC = () => {
   const { filters } = useFilters();
-  let studiesResults = useGetStudiesPageData({ sqon: filters });
+  let studiesResults = useGetStudiesPageData({ sqon: filters, first: 10, offset: 0});
```

Comment:
pagination, for the moment we leave this hardcoded?

Reply:
yes, i'll fix that with real data. I'll fix the UI first